broker/src/test/java/io/camunda/zeebe/broker/system/partitions/impl/NewPartitionTransitionImplTest.java

```diff
@@ -315,12 +328,120 @@ void shouldAbortTransitionIfOneStepThrowsAnExceptionDuringPreparationPhase() {
     verify(mockStep1, never()).transitionTo(mockContext, secondTerm, secondRole);
     verify(mockStep2, never()).transitionTo(mockContext, secondTerm, secondRole);
 
-    assertThatThrownBy(() -> secondTransitionFuture.join())
+    assertThatThrownBy(secondTransitionFuture::join)
         .isInstanceOf(CompletionException.class)
         .getCause()
         .isSameAs(testException);
   }
 
+  @Test // regression test for #8044
+  void shouldCloseAllCreatedInstancesOfStreamProcessor() {
+    // given
+    final var actorScheduler = ActorScheduler.newActorScheduler().build();
+    actorScheduler.start();
+    final var actor = new Actor() {};
+
```

Comment:
As I have written before, I want to keep the changes to the release software as small as possible. Sorry to be uncompromising on this, but I did feel pushed over when we discovered the issue of skipping transitions one week before the release and you insisted on making a last minute fix. And in hindsight, I regret not having insisted on switching the feature flag back to the old code. So because of that I am sticking to my instinct here and will not make any change to a stable branch that is not necessary.

Reply:
I think I should not comment on this further. Just small thing to add: In my opinion bugs can and will happen and this should not fear or block as like this